assets/src/edit-story/components/library/panes/text/textPresets.js

```diff
@@ -31,55 +31,59 @@ const DEFAULT_ELEMENT_WIDTH = PAGE_WIDTH / 2;
 const DEFAULT_LEFT_MARGIN = 40;
 
 const DEFAULT_PRESET = {
-  id: 'default-text',
   content: __('Fill in some text', 'web-stories'),
-  fontSize: dataFontEm(1),
-  font: TEXT_ELEMENT_DEFAULT_FONT,
-  width: 160,
+  fontSize: dataFontEm(1.2),
+  lineHeight: 1.5,
   x: DEFAULT_LEFT_MARGIN,
   y: (PAGE_HEIGHT - dataFontEm(1)) / 2,
```

Comment:
^ Here, too -- should also be adjusted to the new line-height/font-size now.

Reply:
Added line height now 👍

Reply:
Every preset has new line height

Reply:
Oh, I should have been more explicit, sorry.

What I meant was that the `y` value should be adjusted as well based on the new values. The positioning goal is to vertically center the new elements which means it's taking the page height and the element height and dividing it to two for finding the center point. However, now the element height has changed but not in the `y` value.

Reply:
Ah! I see. Let me fix this then.

Reply:
Done now! PTAL!